.github/workflows/deploy-feed-discovery-image.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build and publish a feed-discovery image to ghcr.io
+on:
+  release:
+    types: [published]
+
+  push:
+    branches:
+      - master
```

Comment:
Can you try to change this to the name of the branch you're using and check if this works on your fork?